src/library-authoring/list-libraries/LibraryListPage.jsx

```diff
@@ -207,80 +190,8 @@ export class LibraryListPage extends React.Component {
                     buttonLabels={paginationOptions.buttonLabels}
                     onPageSelect={this.handlePageChange}
                   />
-                )
-                : null}
+                )}
             </article>
-            <aside className="content-supplementary">
```

Comment:
this has been replaced by the AboutLibraraiesHyperLink, right?

Reply:
It isn't here, `AboutLibrariesHyperlink` is inserted at the root of `index.jsx`

Reply:
Yes, so now we don't need this content here. I agree.